### PR TITLE
Modernising as per bincrafters templates

### PR DESCRIPTION
**Pull request by ghost**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -10,13 +10,14 @@ osx: &osx
...
+#        env: CONAN_GCC_VERSIONS=4.9 CONAN_DOCKER_IMAGE=lasote/conangcc49
       - <<: *linux
         env: CONAN_GCC_VERSIONS=5 CONAN_DOCKER_IMAGE=lasote/conangcc5
       - <<: *linux
         env: CONAN_GCC_VERSIONS=6 CONAN_DOCKER_IMAGE=lasote/conangcc6
+      - <<: *linux
+        env: CONAN_GCC_VERSIONS=7 CONAN_DOCKER_IMAGE=lasote/conangcc7
       - <<: *linux
         env: CONAN_CLANG_VERSIONS=3.9 CONAN_DOCKER_IMAGE=lasote/conanclang39
       - <<: *linux
```
Comment (uilianries) [inline .travis.yml:23]: Consider to add clang-5.0

File: .travis.yml [hunk 2]
```diff
@@ -10,13 +10,14 @@ osx: &osx
    language: generic
 matrix:
    include:
-
-      - <<: *linux
-        env: CONAN_GCC_VERSIONS=4.9 CONAN_DOCKER_IMAGE=lasote/conangcc49
+#      - <<: *linux
```
Comment (uilianries) [inline .travis.yml:13]: Why did you remove gcc-4.9?

Comment (ghost) [reply .travis.yml:13]: Because it crashes when setting up xorg-dev as a system dependency. It's the only image that does this.

File: .travis.yml (hunk 1 above)
Comment (ghost) [reply .travis.yml:23]: Done, thanks!

**Review by uilianries — APPROVED** [review]

**Comment by uilianries** [comment]
@dimi309 Thanks!

**Comment by ghost** [comment]
Thank you for merging!

**Comment by uilianries** [comment]
I'll push into stable branch, later

**Comment by ghost** [comment]
ok!
